assets/css/main.css

```diff
@@ -0,0 +1,68 @@
+/********( CSS Index )*********************************************************/
+/*
+    0. CSS Introduction
+    1. General Style
+    2. Javascript Zoom
+    3. Breadcrumbs
+    4. Print Styles
+ */
+
+/********( CSS Introduction )**************************************************/
+/*
+    This file contains generic style rules that are used across all pages.
+*/
+/*=======( General Style )====================================================*/
+.markdown-body {
+    box-sizing: border-box;
+    min-width: 200px;
+    max-width: 980px;
+    margin: 0 auto;
+    padding: 45px;
+    position: relative;
+}
+
+/*=======( Javascript Zoom )==================================================*/
+.js-image-zoom {
+    border: 1px solid #EEE;
+}
+
+/* magnifying glass icon */
+.js-image-zoom::before{
+    background:url(/images/magnifying-glass.png);
```

Comment:
here you don't use a space after the : and in the rest of the file you do. Since we don't have a styleguide for css yet both are technically correct. I think it's more readable with space and I also think you should use be consistent in one file.